base_url/models/abstract_url.py

```diff
@@ -239,6 +239,10 @@ def _sync_urls(self):
 
     @api.model
     def create(self, value):
+        # Creating a binding from the main record set the recompute env to
+        # False and broke the implementation of url sync
+        # need to force it to True
+        self.env.all.recompute = True
```

Comment:
@sebastienbeau I'm pretty sure that this change could lead to performance issue....

Reply:
Hi @lmignon I have review this PR and created a new one.
https://github.com/shopinvader/odoo-shopinvader/pull/607